parameter_files/fates_params_default.cdl

```diff
@@ -1023,7 +1023,8 @@ data:
 
  fates_damage_recovery_scalar = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ;
 
- fates_dev_arbitrary_pft = _, _, _, _, _, _, _, _, _, _, _, _ ;
+ fates_dev_arbitrary_pft = 1.7560, 1.4995, 1.4995, 1.7560, 1.7560, 1.7560, 
+    2.0749, 2.0749, 2.0749, 2.1956, 2.1956, _ ;
```

Comment:
@ckoven to clarify, the last entry here is supposed to be `_`?

Reply:
Yes, it relates to the C4 question. Since we aren't using this scheme for C4 plants at this point, I thought it would be clearest to leave the parameter missing for them as well.

Reply:
Roger that.  Thanks!